win32ss/user/ntuser/simplecall.c

```diff
@@ -372,99 +426,120 @@ NtUserCallOneParam(
+            if (!Window)
+            {
+                Ret = 0;
+                break;
+            }
 
-         RETURN( (DWORD_PTR)IntShowOwnedPopups(Window, (BOOL) Param2));
-      }
+            Ret = (DWORD_PTR)IntShowOwnedPopups(Window, (BOOL)Param2);
+            break;
+        }
 
-      case TWOPARAM_ROUTINE_ROS_UPDATEUISTATE:
-      {
-          WPARAM wParam;
-          Window = UserGetWindowObject((HWND)Param1);
-          if (!Window) RETURN(0);
+        case TWOPARAM_ROUTINE_ROS_UPDATEUISTATE:
+        {
+            WPARAM wParam;
+            Window = UserGetWindowObject((HWND)Param1);
+            if (!Window)
+            {
+                Ret = 0;
+                break;
+            }
 
-          /* Unpack wParam */
-          wParam = MAKEWPARAM((Param2 >> 3) & 0x3,
-                              Param2 & (UISF_HIDEFOCUS | UISF_HIDEACCEL | UISF_ACTIVE));
+            /* Unpack wParam */
+            wParam = MAKEWPARAM((Param2 >> 3) & 0x3,
+                                Param2 & (UISF_HIDEFOCUS | UISF_HIDEACCEL | UISF_ACTIVE));
 
-          RETURN( UserUpdateUiState(Window, wParam) );
-      }
+            Ret = UserUpdateUiState(Window, wParam);
+            break;
+        }
 
-      case TWOPARAM_ROUTINE_SWITCHTOTHISWINDOW:
-         STUB
-         RETURN( 0);
+        case TWOPARAM_ROUTINE_SWITCHTOTHISWINDOW:
+            STUB
+            Ret = 0;
+            break;
 
-      case TWOPARAM_ROUTINE_SETCARETPOS:
-         RETURN( (DWORD_PTR)co_IntSetCaretPos((int)Param1, (int)Param2));
+        case TWOPARAM_ROUTINE_SETCARETPOS:
+            Ret = (DWORD_PTR)co_IntSetCaretPos((int)Param1, (int)Param2);
+            break;
 
-      case TWOPARAM_ROUTINE_REGISTERLOGONPROCESS:
-         RETURN( (DWORD_PTR)co_IntRegisterLogonProcess((HANDLE)Param1, (BOOL)Param2));
+        case TWOPARAM_ROUTINE_REGISTERLOGONPROCESS:
+            Ret = (DWORD_PTR)co_IntRegisterLogonProcess((HANDLE)Param1, (BOOL)Param2);
+            break;
 
-      case TWOPARAM_ROUTINE_SETCURSORPOS:
-         RETURN( (DWORD_PTR)UserSetCursorPos((int)Param1, (int)Param2, 0, 0, FALSE));
+        case TWOPARAM_ROUTINE_SETCURSORPOS:
+            Ret = (DWORD_PTR)UserSetCursorPos((int)Param1, (int)Param2, 0, 0, FALSE);
+            break;
 
-      case TWOPARAM_ROUTINE_UNHOOKWINDOWSHOOK:
-         RETURN( IntUnhookWindowsHook((int)Param1, (HOOKPROC)Param2));
-   }
-   ERR("Calling invalid routine number 0x%x in NtUserCallTwoParam(), Param1=0x%x Parm2=0x%x\n",
-           Routine, Param1, Param2);
-   EngSetLastError(ERROR_INVALID_PARAMETER);
-   RETURN( 0);
+        case TWOPARAM_ROUTINE_UNHOOKWINDOWSHOOK:
+            Ret = IntUnhookWindowsHook((int)Param1, (HOOKPROC)Param2);
+            break;
 
-CLEANUP:
-   TRACE("Leave NtUserCallTwoParam, ret=%p\n", (PVOID)_ret_);
-   UserLeave();
-   END_CLEANUP;
+        default:
+        ERR("Calling invalid routine number 0x%x in NtUserCallTwoParam(), Param1=0x%x Parm2=0x%x\n",
```

Comment:
Missing indent, also `Ret` is uninitialized in the default case.

Reply:
Fixed